Packs/McAfee_ESM/Integrations/McAfee_ESM_v2/McAfee_ESM_v2_test.py

```diff
@@ -246,6 +248,65 @@ def test_edit_case(mocker):
     assert len(result['eventList']) > 0
 
 
+@freeze_time('2022-10-18T16:46:25Z')
+def test_alarm_to_incidents(mocker):
+    """
+    Given: last run object.
+
+    When: running fetch incidents.
+
+    Then: return correct incidents based on alerts time.
+
+    """
+    def fetch_alarams(since: str = None, start_time: str = None, end_time: str = None, raw: bool = False):
+        if type(start_time) == str:
+            start_time = datetime.strptime(start_time, McAfeeESMClient.demisto_format)
+        all_alarms = [alarm for alarm in alarms if
```

Comment:
where is `alarms` defined?

Reply:
under the inner functions

Reply:
a bit confusing, please reorder it for readability